common/security.py

```
@@ -0,0 +1,58 @@
+
+    #DB에 데이터가 있는데 정상적인 숫자가 없으면 1번 시작
+    if not id_numbers:
+        return encode_id(1)
+
+    next_number = max(id_numbers) + 1
+
+    return encode_id(next_number)
+
+def create_access_token(subject:str):
+    minutes = int(os.getenv("ACCESS_TOKEN_EXPIRE_MINUTES"))
+    expire = datetime.utcnow() + timedelta(minutes=minutes)
```

Ruff (0.14.11)

`datetime.datetime.utcnow()` used

(DTZ003)